### PR TITLE
deleted_unnecessary

### PR DESCRIPTION
**Pull request by Yam0r**

### CODE REVIEW

File: src/main/java/mate/academy/springintro/dto/CreateBookRequestDto.java [hunk 1]
```diff
@@ -9,20 +10,22 @@
...
-    @NotNull(message = "Author cannot be null")
+    @NotBlank(message = "Author cannot be null")
     private String author;
 
-    @NotNull(message = "ISBN cannot be null")
+    @NotBlank(message = "ISBN cannot be null")
     private String isbn;
 
-    @NotNull
+    @NotBlank
```
Comment (Elena-Bruyako) [inline src/main/java/mate/academy/springintro/dto/CreateBookRequestDto.java]:
```suggestion
    @NotNull
```

File: src/main/java/mate/academy/springintro/dto/CreateBookRequestDto.java [hunk 2]
```diff
@@ -9,20 +10,22 @@
...
     @Min(value = 0)
     private BigDecimal price;
 
+    @NotNull
```
Comment (Elena-Bruyako) [inline src/main/java/mate/academy/springintro/dto/CreateBookRequestDto.java]:
```suggestion
```

File: src/main/java/mate/academy/springintro/dto/CreateBookRequestDto.java [hunk 3]
```diff
@@ -9,20 +10,22 @@
...
     private String description;
 
+    @NotBlank
```
Comment (Elena-Bruyako) [inline src/main/java/mate/academy/springintro/dto/CreateBookRequestDto.java]:
```suggestion
```

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Why GlobalExceptionHandler is already merged? 
You should provide request with particular HW, delete classes is bad approach

File: src/main/java/mate/academy/springintro/dto/CreateBookRequestDto.java (hunk 3 above)
Comment (Elena-Bruyako) [reply src/main/java/mate/academy/springintro/dto/CreateBookRequestDto.java]: remove annotation

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Don't approve the HW that already merged in main.
Also, the name of Book table should be books not bookshop
Use Lombok annotations instead of generating constructors
There is a mess in PRs so it's better to join QnA session

**Comment by JJJazl** [comment]
Agree with @Elena-Bruyako, will be better to join Q&A session and discuss the task and how to fix this issue. You can also write to your coordinator about it
Don't get upset, we will help you, so you can easily continue your learning journey!)

**Review by sarakhmen — APPROVED** [review]
This pr is approved so please merge it and follow discussed flow further:
- update main branch locally
- create new branch for your homework from the main branch
- add your homework specific changes only and create new PR

**Pull request merged**